### PR TITLE
Feature/more game stats

### PR DESCRIPTION
**Pull request by noahdurbin**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [X] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change

In this branch I worked on more of the game class methods including average goals per game, average goals per season, and the tests to go along with those. refactored some of the other methods to what the spec harness is looking for as well

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before sucmitting PR
- [X] I assigned atleast 2 group members to review my PR

### CODE REVIEW

**Review by austincarrjones — APPROVED** [review]
Solid!

File: lib/game.rb [hunk 1]
```diff
@@ -45,15 +45,23 @@ def self.game_count
   end
 
   def self.percentage_home_wins
-    (count_home_wins.to_f / game_count.to_f) * 100.00
+    (count_home_wins.to_f / game_count.to_f).round(2)
```
Comment (zach-bergman) [inline lib/game.rb:48]: Awesome, should be passing spec harness now

File: lib/game.rb [hunk 2]
```diff
@@ -85,4 +93,20 @@ def self.count_ties
...
+    end / game_count.round(2)
+  end
+
+  def self.average_goals_per_season
+    goals_per_season = Hash.new(0)
+    all_games.each do |game|
+      goals_per_season[game.season] += game.total_score
+    end
+    goals_per_season.each do |season, total_goals|
+      goals_per_season[season] = (total_goals / count_of_games_by_season[season].to_f).round(2)
```
Comment (zach-bergman) [inline lib/game.rb:109]: Really like this, using the count_of_games_by_season to help out

**Review by zach-bergman — APPROVED** [review]
Looks great, approving now

**Pull request merged**